app/models/search_result.rb

```diff
@@ -3,5 +3,5 @@
 class SearchResult
   include ActiveModel::API
   attr_accessor :title, :format, :icon, :physical, :author, :journal, :imprint, :description,
-                :online_badge, :link, :thumbnail, :fulltext_link_html
+                :online_badge, :link, :thumbnail, :fulltext_link_html, :stanford_only_link
```

Comment:
Can we make this a `stanford_only?` method?

Reply:
in the attr_accessor? I don't think so. The original way I did this was with a ? but it broke the code.

Reply:
I would like to include something about the link. The record isn't stanford_only just that one full_text_link but otherwise this is doable.

Reply:
I went with stanford_only_fulltext? I can change it if you prefer but I think fulltext as all one word is more inline with what we already have.